analyzers/its/expected/Ember-MM/scraper.EmberCore-{EF6A550E-DD76-4F4D-8250-8598140F828B}-S3900.json

```diff
@@ -56,6 +56,19 @@
 "id":  "S3900",
 "message":  "Refactor this method to add validation of parameter 'imgResult' before using it.",
 "location":  {
+"uri":  "https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/its/sources/Ember-MM/Addons/scraper.EmberCore/Scraper/clsScrapeImages.vb#L435",
+"region":  {
+"startLine":  435,
+"startColumn":  9,
+"endLine":  435,
+"endColumn":  18
+}
+}
+},
```

Comment:
I believe this issue (TP) was not found before, because execution reached its step limit. This PR cuts some branches, which allows the engine to explore further and find this issue.

Reply:
Yes! Look here:https://github.com/SonarSource/sonar-dotnet/pull/7884/files#diff-f850fe1a39a7c94f77702894b681861040424baec8ca0f046a831d4ad14793f8R59
After the change Pavel suggested it appears here too!